src/js/state/Spaces/reducer.js

```diff
@@ -15,7 +16,20 @@ const spacesReducer = produce((draft, action: SpacesAction) => {
 
     case "SPACES_DETAIL":
       var {name} = action.space
-      draft[name] = {...draft[name], ...action.space}
+      var space = (action.space: any)
```

Comment:
Makes sense to support both. Are you willing to move this logic to a new function in `src/js/brim/interop.js` then call it in here?

That's were functions go to convert between old and new types.

Reply:
sure, make sense